ExpenseTracker/ExpenseTracker/Views/Notifications/Index.cshtml

```diff
@@ -175,7 +174,6 @@
         border-color: #d1d5da;
     }
 
-    /* Popup Modal Styles */
     .popup-modal {
         display: none;
         position: fixed;
```

Comment:
I didn't tell you about unnecessary comments. I told you about that you have written styles for modal

Reply:
Site.css has styles for all modal-popups.
You shouldn't write again this styles.

Reply:
I understand but for now I'm using this because I'm not using site.css and I'll keep this in mind when I want to add css styles to site.css later